libs/styles/styles.css

```diff
@@ -316,6 +316,12 @@
   text-decoration: none;
 }
 
+.con-button:focus-visible,
+button:focus-visible {
```

Comment:
The idea was to implement a uniform accessibility functionality in milo. The buttons tabbing was not implemented in MILO

Reply:
This is a bit heavy handed related to modals. Is there a request to change for all buttons? The Jira ticket only mentions the V2 Geo Modal. 

This should probably be scoped to `.dialog-modal.locale-modal-v2` unless otherwise noted or asked for.  

@ryanmparrish  have you heard anything from Consonant about changing this for ALL `con-buttons`?